drivers/mmc/host/bcm2835-sdhost.c

```diff
@@ -1200,9 +1201,11 @@ static void bcm2835_sdhost_finish_command(struct bcm2835_host *host,
 
 		/* Wait max 100 ms */
 		wait_max = jiffies + msecs_to_jiffies(100);
+		usleep_min = 1;
 		while (time_before(jiffies, wait_max)) {
 			spin_unlock_irqrestore(&host->lock, *irq_flags);
-			usleep_range(1, 10);
+			usleep_range(usleep_min, usleep_min * 8UL);
+			usleep_min = max(256UL, usleep_min * 4UL);
```

Comment:
If you want to cap the value at 256 you need to use min here.
And remember the value is multiplied by 8 so this will be 2048us.